shop/urls.py

```diff
@@ -1,10 +1,16 @@
 from django.contrib import admin
 from django.urls import path,include
 from django.contrib.auth import views as auth_views
+from django.conf import settings
+from django.conf.urls.static import static
```

Comment:
These are imports that are enabling us to set where we want to get the media files during development.